kotlin-data-wedge-lib/src/main/java/dk/gls/kdw/configuration/scanner/DeviceId.kt

```diff
@@ -0,0 +1,28 @@
+package dk.gls.kdw.configuration.scanner
+
+enum class DeviceId {
```

Comment:
Do we need to add each device we support here?

Reply:
I guess.